### PR TITLE
Removed rounded corners and some other minor fixes

### PR DESCRIPTION
**Pull request by felixoi**
Screenshot from Gyazo

Hope this makes some guys happy :100:

### CODE REVIEW

File: app/assets/stylesheets/main.scss [hunk 1]
```diff
@@ -275,13 +281,12 @@ form {
 }
 
 select.form-control, input.form-control {
-  border-radius: 4px;
   box-shadow: none;
-  border: 1px solid $sponge_grey;
+  border: 1px solid #ddd;
```
Comment (phase) [inline app/assets/stylesheets/main.scss]: I'd prefer these be in variables.

**Review by phase — CHANGES_REQUESTED** [review]
Looks good so far.

File: app/assets/stylesheets/main.scss (hunk 1 above)
Comment (progwml6) [reply app/assets/stylesheets/main.scss]: all colors should be in variables

**Comment by Grinch** [comment]
Hallelujah!

**Review by Ewoutvans — APPROVED** [review]

**Pull request merged**